Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -10,7 +10,7 @@
   "version": {
     "Major": 0,
     "Minor": 1,
```

Comment:
change minor version to 128 and patch to 0. we can follow same pattern for both the download tasks.

Reply:
changed